packages/esm-patient-search-app/src/patient-search/patient-search.resource.tsx

```diff
@@ -1,7 +1,7 @@
 import { openmrsFetch } from '@openmrs/esm-framework';
 
 export function performPatientSearch(query: string, objectVersion: string, ac: AbortController, includeDead: boolean) {
-  const url = `/ws/rest/v1/patient?q=${query}&v=${objectVersion}&includeDead=${includeDead}`;
+  const url = `/ws/fhir2/R4//Patient?name:contains=<q>&identifier=<q>`;
```

Comment:
So at a minimum we need this:

```suggestion
  const url = `/ws/fhir2/R4//Patient?name:contains=${query}&identifier=${query}`;
```

We also need to handle the `includeDead` parameter, which is a little weird. `includeDead` means include both deceased and living people, so we should have something like:

```ts
`${includeDead ? '&deceased=true,false' : ''}
```

Also, requests to the FHIR API should usually have the parameter `&_summary=data` at the end to avoid having to generate the FHIR narrative.

Maybe to handle that. The biggest part of this ticket, however, it to update the places that call `performPatientSearch` to ensure they continue to work when we switch to using the FHIR API, since FHIR API data is in a different format from data from the REST API.